packages/blocks/src/root-block/root-service.ts

```diff
@@ -411,4 +401,22 @@ export class RootService extends BlockService<RootBlockModel> {
     assertExists(viewportElement);
     return viewportElement;
   }
+
+  abstract slots: {
```

Comment:
We have a `RefNodeSlots` type in `@blocksuite/affine-components/rich-text`

Reply:
Got it. Thanks for your comments!